data-prepper-plugins/otel-trace-raw-prepper/src/test/resources/sample-request-one-full-trace-group.json

```diff
@@ -0,0 +1 @@
+{"resourceSpans":[{"resource":{"attributes":[{"key":"service.name","value":{"stringValue":"analytics-service"}},{"key":"telemetry.sdk.language","value":{"stringValue":"java"}},{"key":"telemetry.sdk.name","value":{"stringValue":"opentelemetry"}},{"key":"telemetry.sdk.version","value":{"stringValue":"0.8.0-SNAPSHOT"}}],"droppedAttributesCount":0},"instrumentationLibrarySpans":[{"instrumentationLibrary":{"name":"io.opentelemetry.auto.spring-webmvc-3.1","version":""},"spans":[{"traceId":"/6V20yEXOsbO82Acj0vedQ==","spanId":"CFrAgv/Pv40=","traceState":"","parentSpanId":"yxwHNNFJQP0=","name":"LoggingController.save","kind":"SPAN_KIND_INTERNAL","startTimeUnixNano":"1597902043168792500","endTimeUnixNano":"1597902043215953100","attributes":[],"droppedAttributesCount":0,"events":[],"droppedEventsCount":0,"links":[],"droppedLinksCount":0,"status":{"code":"STATUS_CODE_OK","message":""}}]},{"instrumentationLibrary":{"name":"io.opentelemetry.auto.apache-httpasyncclient-4.0","version":""},"spans":[{"traceId":"/6V20yEXOsbO82Acj0vedQ==","spanId":"G4PRdsC1+0Y=","traceState":"","parentSpanId":"CFrAgv/Pv40=","name":"HTTP PUT","kind":"SPAN_KIND_CLIENT","startTimeUnixNano":"1597902043175204700","endTimeUnixNano":"1597902043205117100","attributes":[{"key":"http.status_code","value":{"intValue":"200"}},{"key":"http.url","value":{"stringValue":"/logs/_doc/service_1?timeout\\u003d1m"}},{"key":"http.method","value":{"stringValue":"PUT"}}],"droppedAttributesCount":0,"events":[],"droppedEventsCount":0,"links":[],"droppedLinksCount":0,"status":{"code":"STATUS_CODE_OK","message":""}}]},{"instrumentationLibrary":{"name":"io.opentelemetry.auto.servlet-3.0","version":""},"spans":[{"traceId":"/6V20yEXOsbO82Acj0vedQ==","spanId":"yxwHNNFJQP0=","traceState":"","parentSpanId":"","name":"/logs","kind":"SPAN_KIND_SERVER","startTimeUnixNano":"1597902043168010200","endTimeUnixNano":"1597902043217170200","attributes":[{"key":"http.status_code","value":{"intValue":"200"}},{"key":"net.peer.port","value":{"intValue":"41164"}},{"key":"servlet.path","value":{"stringValue":"/logs"}},{"key":"http.response_content_length","value":{"intValue":"7"}},{"key":"http.user_agent","value":{"stringValue":"curl/7.54.0"}},{"key":"http.flavor","value":{"stringValue":"HTTP/1.1"}},{"key":"servlet.context","value":{"stringValue":""}},{"key":"http.url","value":{"stringValue":"http://0.0.0.0:8087/logs"}},{"key":"net.peer.ip","value":{"stringValue":"172.29.0.1"}},{"key":"http.method","value":{"stringValue":"POST"}},{"key":"http.client_ip","value":{"stringValue":"172.29.0.1"}}],"droppedAttributesCount":0,"events":[],"droppedEventsCount":0,"links":[],"droppedLinksCount":0,"status":{"code":"STATUS_CODE_OK","message":""}}]}]}]}
```

Comment:
nit - if possible would be good to auto-format these JSON files. Mostly so other developers can easily understand the structure and contents of the payload.